functions/GSEClause.cc

```diff
@@ -114,15 +115,14 @@ GSEClause::set_start_stop()
     int i = d_start;
     int end = d_stop;
     while (i <= end && !compare<T>(vals[i], d_op1, d_value1))
-        i++;
-
+        ++i;
     d_start = i;
 
-    // Now scan backward from the end. We scan all the way to the actual start
+    // Now scan backward from the end. We scan all the way to the actual start,
     // although it would probably work to stop at 'i >= d_start'.
     i = end;
     while (i >= 0 && !compare<T>(vals[i], d_op1, d_value1))
-        i--;
+        --i;
```

Comment:
In this line, this is a completely pointless change. If it was in some other evaluation say,

`   foo.method(i--);`
vs 
`   foo.method(--i);`

It would be meaningful. Here it is not.